notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<p>This cell is being repeated four times in the notebook. I suggest converting it into a function. </p><p>I use the DRY (don't repeat yourself). If you repeat a code three times, it is better to write a function for it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I agree, but it happens when I try to convert it to a function. I get an error related to the 'await' command, which doesn't retrieve any data. I have never used it before, I will explored deeply into this. Do you have any tips for me?</p>

Reply:
<p>From now on, the data is download at once, and then it is processed by the script.</p>

Reply:
There is another option to query EFD data without using async functions. Try this:
```
from lsst.summit.utils.efdUtils import makeEfdClient, getEfdData
```
And, then, something like this:
```
start = Time("2024-03-09T10:00:00Z", scale="utc", format="isot")
end = Time("2024-03-11T06:00:00Z", scale="utc", format="isot")

df_az = getEfdData(
    client,
    "lsst.sal.MTMount.azimuth",
    columns=["actualPosition", "actualTorque"],
    begin=start,
    end=end,
)
```
This should help you wrapping this part into a function.

Reply:
wau! that will help a lot. :)